### PR TITLE
Update and rename checkov.yml to kics.yml

### PR DESCRIPTION
**Pull request by the-tatanka**

### CODE REVIEW

File: .github/workflows/kics.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+name: "KICS"
+
+on:
+  push:
+    branches: [main, master]
+  # pull_request:
```
Comment (mkanal) [inline .github/workflows/kics.yml:6]: According to internal team aggreement kics should be executed on each pull request.

File: .github/workflows/kics.yml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  # The branches below must be a subset of the branches above
+  # branches: [main, master]
+  # paths-ignore:
+  #   - "**/*.md"
+  #   - "**/*.txt"
+  schedule:
+    - cron: "28 15 * * 3"
```
Comment (mkanal) [inline .github/workflows/kics.yml:13]: Why At 15:28 on Wednesday?

Comment (mkanal) [reply .github/workflows/kics.yml:13]: please add comment when cron is executed.

**Review by mkanal — CHANGES_REQUESTED** [review]

File: .github/workflows/kics.yml (hunk 1 above)
Comment (the-tatanka) [reply .github/workflows/kics.yml]: Done

File: .github/workflows/kics.yml (hunk 2 above)
Comment (the-tatanka) [reply .github/workflows/kics.yml]: Oh no, my mistake.
Thanks for the hint.

**Review by mkanal — APPROVED** [review]

**Pull request merged**